bing.com.BingWebMasterDPOO.json

```diff
@@ -0,0 +1,22 @@
+{
+  "providerId": "bing.com",
+  "providerName": "Microsoft Bing",
+  "serviceId": "BingWebMasterDPOO",
+  "serviceName": "Domain proof of ownership with Bing Webmaster services",
+  "version": 2,
+  "logoUrl": "https://domainconnect.org/wp-content/uploads/<<bing logo>> link",
+  "description": "Verifies domain ownership for further configuration of Microsoft Bing Services",
+  "variableDescription": "Guid of webmaster for verification of domain ownership with Bing webmaster enables quick and easy SEO on Bing.",
+  "syncBlock": false,
+  "shared": false,
+  "warnPhishing": false,
+  "syncRedirectDomain": "www.bing.com",
+  "records": [
+      {  
+         "type":"CNAME",
+         "host":"%WEBMASTERGUID%",
```

Comment:
In general we aren't super excited about variables in the host field.  However, the spec does make an exception for a cname used for validation.  So I'm OK with it.

Reply:
We need this to be variable, we notify users the actual value of this field before making DNS changes. So, this should be okay to keep host as variable.